tests/test_custom_checkers/test_pycodestyle_checker.py

```diff
@@ -39,6 +45,7 @@
     ("E301", [(5, 4, "expected 1 blank line, found 0")]),
     ("E303", [(6, 0, "too many blank lines (3)")]),
     ("E304", [(12, 0, "blank lines found after function decorator")]),
+    ("E306", [(3, 4, "expected 1 blank line before a nested definition, found 0")]),
```

Comment:
@shirleyyzhang sorry I missed this in my initial review, but I don't think you intended to add this line (as there are no example files for this error)